app/src/main/java/com/example/moodtrackerproject/ui/mood/list/MoodViewModel.kt

```diff
@@ -35,10 +35,31 @@ class MoodViewModel : BaseViewModel<MoodProps>() {
             openStressTestScreen = {
                 setState(state, action = MoodScreenActions.StartStressTestScreen)
             },
-            fetchListOfMoods = ::fetchListOfMoods
+            openAnxietyTestScreen = {
+                setState(state, action = MoodScreenActions.StartStressTestScreen)
+            },
+            fetchListOfMoods = ::fetchListOfMoods,
+            setTestType = ::setTestType
         )
     }
 
+    private fun setTestType(type: Int) {
+        val state = Store.appState.stressTestState
+        if (type == 1) {
```

Comment:
Refactor to 
`Store.setState(
              state.copy(
                    questionList = if (type == 1) DataBaseRepository.listOfAnxietyQs else DataBaseRepository.listOfStressQs
                )
            )`